release/gcb/store_artifacts.sh

```diff
@@ -28,5 +32,8 @@ source "/workspace/gcb_env.sh"
 
 OUTPUT_PATH="/output"
 
+# create the istioctl release archives
+"${ROOT}"/release/create_release_archive.sh -o "${OUTPUT_PATH}" -v "${VER_STRING}"
```

Comment:
create_release_archive is already called from release/gcb/cloud_builder.sh.

Reply:
@utka does this mean that the istioctl binaries are already built? If so, are they store somewhere other than this output directory `/output`?